packages/@n8n/nodes-langchain/nodes/memory/MemoryBufferWindow/MemoryBufferWindow.node.ts

```diff
@@ -130,13 +130,7 @@ export class MemoryBufferWindow implements INodeType {
 				},
 			},
 			sessionKeyProperty,
-			{
-				displayName: 'Context Window Length',
-				name: 'contextWindowLength',
-				type: 'number',
-				default: 5,
-				description: 'The number of previous messages to consider for context',
-			},
+			contextWindowLengthProperty(),
```

Comment:
Just a small note: maybe a simple object would do here? and in case it will be necessary - extend it like this:```
{
  ...contextWindowLengthProperty,
  displayOptions: something
}
```

Reply:
I felt that that was a bit more cumbersome in the base-case, but maybe you're right. I'll check it out, thanks.